wgengine/router/callback.go

```diff
@@ -41,6 +49,10 @@ func (r *CallbackRouter) Set(rcfg *Config) error {
 	if r.rcfg.Equal(rcfg) {
 		return nil
 	}
+	if r.initialMTU != 0 {
```

Comment:
(this will always be `0` if `r.initialMTU` is never initialized)

Reply:
Fixed.